<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>liaohw</author>
 <class>MyForm</class>
 <widget class="QWidget" name="MyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编码格式批量转换</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>路径：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TransferButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>按制定格式进行转换！</string>
   </property>
   <property name="text">
    <string>执行转换</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>请输入路径</string>
   </property>
   <property name="text">
    <string>g:\test</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>621</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>41</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>过滤：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_split">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>41</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>默认过滤：.zip\.rar\.tar</string>
   </property>
   <item>
    <property name="text">
     <string>.svn</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.cache</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>41</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>源格式：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_ori">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>41</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>utf-8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GB2312</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ascii</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>utf_8_sig</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gbk</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>windows-1252</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>*</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>41</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>新格式：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_new">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>41</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>GB2312</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ascii</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>utf-8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>utf_8_sig</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gbk</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>windows-1252</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="DetectButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>检测路径下所有文件类型</string>
   </property>
   <property name="text">
    <string>检测类型</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
 <slots>
  <slot>ftpCon()</slot>
  <slot>runTransfer()</slot>
  <slot>change()</slot>
 </slots>
</ui>
